be/src/exec/pipeline/fragment_executor.cpp

```diff
@@ -1045,11 +1045,18 @@ Status FragmentExecutor::_decompose_data_sink_to_operator(RuntimeState* runtime_
         RETURN_IF_ERROR(
                 Expr::create_expr_trees(runtime_state->obj_pool(), output_exprs, &output_expr_ctxs, runtime_state));
 
+        int64_t max_file_size = 1024 * 1024 * 1024L;
```

Comment:
could it be a config item? it's the max file size if it's not specified in FE.

Reply:
it is unnecessary... this PR has provided a more flexible way.

Reply:
yes, but better to define this value in a explicitly way. how to know it's 1G if max file size is not specified by FE?